<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>555</width>
    <height>395</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="id">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>56</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>아이디</string>
   </property>
  </widget>
  <widget class="QLabel" name="pw">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>130</y>
     <width>56</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>비밀번호</string>
   </property>
  </widget>
  <widget class="QLabel" name="name">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>160</y>
     <width>56</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>이름</string>
   </property>
  </widget>
  <widget class="QLabel" name="address">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>190</y>
     <width>56</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>주소</string>
   </property>
  </widget>
  <widget class="QLabel" name="phone">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>220</y>
     <width>56</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>휴대폰</string>
   </property>
  </widget>
  <widget class="QLabel" name="e_mail">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>250</y>
     <width>56</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>이메일</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>340</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>가입</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>100</y>
     <width>75</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>중복체크</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="id_line">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>100</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="pw_line">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>130</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="name_line">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>160</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="address_line">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>190</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="phone_line">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>220</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="e_mail_line">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>250</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>회원가입</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>id_check()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>370</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>483</x>
     <y>81</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>sign()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>241</x>
     <y>351</y>
    </hint>
    <hint type="destinationlabel">
     <x>459</x>
     <y>349</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>id_check()</slot>
  <slot>sign()</slot>
 </slots>
</ui>
